notebooks/266-table-question-answering/266-table-question-answering.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<p>&gt;&gt;In this tutorial we demonstrate how to use <a href="https://huggingface.co/tasks/table-question-answering" target="_blank">this</a> base example with OpenVINO.</p><p>it is not clear what mean this. I think better to say:</p><p> </p><p>In this tutorial we demonstrate how to perform table question answering using OpenVINO</p><p> </p><p>The header is very specific and detailed (you lost that it is about table question answering and replacement model in tutorial will also required to change header), maybe better to name it more general: Table Question Answering using TAPAS and OpenVINO</p><p> </p><p>or Transformer-based Table Question Answering with OpenVINO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
fixed